features/facebookImport/src/model/importers/multiple-files-importer.js

```diff
@@ -41,51 +40,48 @@ export default class MultipleFilesImporter extends Importer {
             });
     }
 
-    async _importDataFromFiles(targetEntries, zipFile, facebookAccount) {
-        const fileDataWithResults = await Promise.all(
-            targetEntries.map((targetEntry) =>
-                this._readJSONFileWithStatus(targetEntry, zipFile)
-            )
-        );
-
-        const successfullResults = fileDataWithResults.filter(
-            (result) => result.status.isSuccess
-        );
-
-        for (const each of successfullResults) {
-            facebookAccount.addImportedFileName(each.targetEntry.path);
-        }
-        const dataResults = successfullResults.map((result) => result.data);
-        this._importRawDataResults(facebookAccount, dataResults);
-
-        return fileDataWithResults
-            .filter((result) => !result.status.isSuccess)
-            .map(({ status }) => status);
-    }
-
     _createMissingFilesError() {
         return new MissingFilesException();
     }
 
-    async import({ zipFile, facebookAccount }) {
+    async import({ zipFile }) {
```

Comment:
how about refactoring `import` (~40LOC) into smaller funcs , as its nesting length and current understandability are quite tech-debt-proned ...we could be made slightly easier to understand